R/pkg/R/generics.R

```diff
@@ -507,6 +507,10 @@ setGeneric("saveAsTable", function(df, tableName, source, mode, ...) {
   standardGeneric("saveAsTable")
 })
 
+#' @rdname withColumn
+#' @export
+setGeneric("transform", function(`_data`, ...) {standardGeneric("transform") })
```

Comment:
that's the signature from the original definition. please see discussion above.
local data.frame or vector will fail to run with transform if this doesn't match exactly.